src/elli.erl

```diff
@@ -129,11 +129,13 @@ init([Opts]) ->
     MinAcceptors   = proplists:get_value(min_acceptors, Opts, 20),
 
     UseSSL         = proplists:get_value(ssl, Opts, false),
```

Comment:
It seems the test is failing on this boolean.

Reply:
It shouldn't fail as a result of my change, maybe something else?